lib/sort_by_length.rb

```diff
@@ -1,7 +1,25 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2 * m) since bubble sort uses the .length method. N is number of words in input string. M is average number of characters in each word.
```

Comment:
Nice bubble sort.  What would you need to change to make this a "smart" bubble sort?